.env.local

```diff
@@ -13,3 +13,13 @@ LOGSTASH_PACKAGE_PATH=
 JS_REPORT_DEV_MOUNT=false
 JS_REPORT_PACKAGE_PATH=
 
+CLUSTERED_MODE=false
+
+MPI_CLIENT_ID=DISI CLIENT
+MPI_CLIENT_SECRET=DISI
+
+# Disables Fhir validation agains IG
+DISABLE_VALIDATION=true
+
+# Name of folder containing the Fhir IG importer scripts in the importer folder
+FHIR_CONFIG_IMPORTER_FOLDER_NAME=hapi-fhir
```

Comment:
We can remove this from the `.env` var files if we have the default in `package-metadata` anyway.